src/patterns/atoms/typographic/quotes.hbs

```diff
@@ -0,0 +1,14 @@
+---
+name: Quote Elements
+description: |
+  A pull quote is an excerpt that has been pulled from a body of text used to highlight a key topic. A blockquote is a quotation that is set off from the main text typically used for long quotations.
+links:
+  "MDN <blockquote>": https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote
+order: 5
+---
+
+<blockquote class="mzp-c-quote">
+  <p>
+    "Chupadados is the 21st century iteration, more modern and more cute," Varon said. "It means 'date sucker,'"
```

Comment:
If you like, you can use `{{data "specimens.sentence"}}` to get a sample sentence here.
https://github.com/mozilla/protocol/blob/master/src/data/specimens.yaml